packages/dtslint/src/index.ts

```diff
@@ -138,6 +138,7 @@ async function runTests(
     // so assert that we're really on DefinitelyTyped.
     assertPathIsInDefinitelyTyped(dirPath);
     assertPathIsNotBanned(dirPath);
+    assertPackageIsNotDeprecated(dirPath, await readFile("../notNeededPackages.json", "utf-8"));
```

Comment:
@andrewbranch I just cloned DT. trying to update a type package. When running `npm t <packageName>` I got:
```
Error: ENOENT: no such file or directory, open '../notNeededPackages.json'
```
Debugging revealed this to be the line that throws.

When calling `readFile`, if using a relative path as a parameter, it will use current cwd to resolve that to an absolute path. This might be right for specific cases where `notNeededPackage.json` is indeed just one level up the cwd, but breaks running dtslint from the root of DefinitelyTyped.

TNH, I'm not sure how there was no additional noise regarding this. Nobody executes tests locally? Or perhaps they are still running older dtslint? Or maybe this change took time to get released..?

Reply:
@AviVahl fwiw, I happen to have just hit the same thing. #402

Reply:
Thanks for the sleuthing @AviVahl. I'll see what I can do.

> Nobody executes tests locally? 

I would guess this is it.

Reply:
Thanks for the fix :+1:

Reply:
Fix is up at #405